entgql/template/pagination.tmpl

```diff
@@ -311,28 +319,37 @@ func (p *{{ $pager }}) applyFilter(query *{{ $query }}) (*{{ $query }}, error) {
 
 {{ $r := $node.Receiver -}}
 func (p *{{ $pager }}) toCursor({{ $r }} *{{ $name }}) Cursor {
-	return p.order.Field.toCursor({{ $r }})
+	return p.orders[0].Field.toCursor({{ $r }})
```

Comment:
I have some confuse right here. Should the cursor holding information about all orders to do the right pagination?